### PR TITLE
STR-250 -- FE: Refactor informational template

### PR DESCRIPTION
**Pull request by BrianLinggadjaja**
# Description
Decoupled shared module templates and re-organized file/folder structure.

# Testing
1. Check to see if all previous functionality for templates works.
2. Check to ensure all video templates are properly placed.
3. Ensure all slide template folder/file structure makes sense.

### CODE REVIEW

File: resources/js/components/module/templates/shared/VideoPlayback.vue [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="video-template justify-content-center mb-5 row">
+    <div class="video-template__video col-12 col-md-10">
```
Comment (JaimeGG64) [inline resources/js/components/module/templates/shared/VideoPlayback.vue]: Add a `p-0` so the video and take the full on mobile view.

Comment (BrianLinggadjaja) [reply resources/js/components/module/templates/shared/VideoPlayback.vue]: Resolved

**Review by JaimeGG64 — DISMISSED** [review]
If you can add a `width: 100%` for the `module__wrapper` class.

**Review by JaimeGG64 — APPROVED** [review]
💵

**Pull request merged**